### PR TITLE
Better layout configs. issue  #1300  solved

### PR DESCRIPTION
**Pull request by prachi237**
Changes suggested are done. readme docs is now in its updated version. merging this PR will close the issue #1300 

PR Checklist:

- [x] make all edits to the docs in the `src` directory, not in `docs`
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] put any other relevant information below

### CODE REVIEW

**Comment by prachi237** [comment]
Screenshot (89)

@viniciusdc  sir completed!

File: src/README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
-To build the docs here locally: make sure you have Sphinx, the Cloud Sphinx Theme, and the sphinxcontrib-fulltoc Sphinx extension installed (`conda install sphinx cloud_sptheme sphinxcontrib-fulltoc`), then run `make html` (Unix) or `./make.bat html` (Windows). This will create the output files in `_build/html`.
+To build the docs here locally: 
+* Make sure you have Sphinx, the Cloud Sphinx Theme, and the sphinxcontrib-fulltoc Sphinx extension installed (``conda install sphinx cloud_sptheme sphinxcontrib-fulltoc``).
+* Then run ``make html`` (Unix) or ``./make.bat html`` (Windows).
+* This will create the output files in ``_build/html``.
```
Comment (beckermr) [inline src/README.md:4]: Let's remove this section.

Comment (prachi237) [reply src/README.md:4]: > Let's remove this section.

okie sir

Comment (prachi237) [reply src/README.md]: @beckermr done sir

**Comment by beckermr** [comment]
LGTM! Will merge once the CI passes.

**Pull request merged**